### PR TITLE
chore: add managed dependencies

### PR DESCRIPTION
**Pull request by kweinmeister**
Added `libraries-bom` support to samples missing it in `pom.xml`, and where at least 1 dependency would be used from the BOM.

### CODE REVIEW

**Comment by kweinmeister** [comment]
[snippet-bot check failure](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7894#issuecomment-1509896707) is due to an existing region tag issue.

File: flexible/java-8/websocket-jsr356/pom.xml [hunk 1]
```diff
@@ -46,7 +59,8 @@ limitations under the License.
       <type>jar</type>
       <scope>provided</scope>
     </dependency>
-    <!--Jetty JSR-356 Websocket client side dependency-->
+    <!--Jetty
```
Comment (Shabirmean) [inline flexible/java-8/websocket-jsr356/pom.xml:62]: Why the line breaks here? Was it automatic? Can we put them in the same line?

File: flexible/websocket-jsr356/pom.xml [hunk 2]
```diff
@@ -46,7 +59,8 @@ limitations under the License.
       <type>jar</type>
       <scope>provided</scope>
     </dependency>
-    <!--Jetty JSR-356 Websocket client side dependency-->
+    <!--Jetty
```
Comment (Shabirmean) [inline flexible/websocket-jsr356/pom.xml:62]: Same here. Can we remove these line breaks?

File: media/livestream/pom.xml [hunk 3]
```diff
@@ -36,7 +36,8 @@
     <maven.compiler.source>11</maven.compiler.source>
   </properties>
   <!--  Using libraries-bom to manage versions.
-  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  See
+  https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
```
Comment (Shabirmean) [inline media/livestream/pom.xml:40]: (nit) Move "See" to the previous line

File: media/stitcher/pom.xml [hunk 4]
```diff
@@ -36,7 +36,8 @@
     <maven.compiler.source>11</maven.compiler.source>
   </properties>
   <!--  Using libraries-bom to manage versions.
-  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  See
```
Comment (Shabirmean) [inline media/stitcher/pom.xml:39]: Same (nit) as before

**Review by Shabirmean — APPROVED** [review]
LGTM! Minor nits

**Pull request merged**